### DISCUSSION TITLE
How to submit issue for non-functional change?

### DISCUSSION THREAD

**Discussion by krono** [netbox-community/netbox#6201] [Q&A] [answered]
I'd like to contribute a patch to netbox but don't know where to put it right.

I think that a certain part of the Network Interface descriptions would benefit from a change, with a small few-lines patch. (in nuce: remove bandwidth indicators from InfiniBand interfaces for good reason).

I understand that PRs must be preceeded by accepted issues, but the Issue I want to submit does not seem to fit the "Bug" category (it is a non-functional change in descriptions only) nor a "Feature Request" (since I'd already have a proposed solution), nor "Documentation Change" (since it pertains code), nor "Housekeeping".

So, where to put it? Maybe it's just a simple language barrier thing for me, but I somehow missing an "everything else" category in the issue creation?

**Comment by candlerb** [comment, marked as answer]
It's a "Feature Request".  In that ticket, you describe the change you want to make (from the user's point of view).

You then create a separate pull request with the patch, and reference the feature request issue number.

**Reply by krono** [reply]
That makes sense, thanks.

**Reply by krono** [reply]
Well, that apparently was not well-received:

https://github.com/netbox-community/netbox/issues/6303

:(

**Reply by krono** [reply]
Also, thes proposed course of action is apparenlty not to have a pull request…
https://github.com/netbox-community/netbox/pull/6304

**Reply by candlerb** [reply]
It said "there is no _accepted issue_ that correlates to this pull request".  That is: if the FR hasn't been accepted as a desirable change, then the PR that implements that FR isn't merged.

**Reply by krono** [reply]
Thanks for pointing that out.

I wonder, so there is no intention to back up an FR with a PoC or even a change that would work?

somehow this experience was less fun than I expected

**Reply by candlerb** [reply]
I don't think that the existence of a PoC affects whether the feature is desirable or not.

**Reply by krono** [reply]
It affects the cost/benefit ratio, I'd say.
